app/api/chemotion/report_api.rb

```diff
@@ -198,5 +198,146 @@ class ReportAPI < Grape::API
         excel.generate_file
       end
     end
+
+    resource :multiple_reports do
+      desc "Build a multi-reactions report using the contents of a JSON file"
+
+      params do
+        requires :ids
+        requires :settings
+      end
+
+      get :rtf do
+        ids = params[:ids].split("_")
+        settings = params[:settings].split("_")
+
+        report = Report::RTFReport.new do |r|
```

Comment:
same could be said for `get :rtf` from line 9.
This is just following the practice.
If this is to be cleaned so should be the upper part.
I don't mind it for now as long as,  the whole report generation part is moved to, say, somewhere  in lib/report/ with a new issue (next iteration?)